pkg/management/component.go

```diff
@@ -1351,20 +1383,30 @@ func (c *component) GetRoles(ctx context.Context, realmName string) ([]api.RoleR
 
 	var rolesRep = []api.RoleRepresentation{}
 	for _, roleKc := range rolesKc {
-		var roleRep api.RoleRepresentation
-		roleRep.ID = roleKc.ID
-		roleRep.Name = roleKc.Name
-		roleRep.Composite = roleKc.Composite
-		roleRep.ClientRole = roleKc.ClientRole
-		roleRep.ContainerID = roleKc.ContainerID
-		roleRep.Description = roleKc.Description
-
-		rolesRep = append(rolesRep, roleRep)
+		if c.isBusinessRole(roleKc) {
```

Comment:
Factorize? You can write convert functions in api/management

Reply:
Good catch ! Done !